openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiApplicationVisitorSpec.groovy

```diff
@@ -918,7 +899,6 @@ class MyBean {}
 ''')
         then: "the state is correct"
         !Utils.testReference
-        !Utils.testReferenceAfterPlaceholders
```

Comment:
Utils.testReferenceAfterPlaceholders - this filed deleted and we need to check testReference always